roles/confluent.common/tasks/debian.yml

```diff
@@ -28,3 +28,16 @@
 
 - import_tasks: debian_jvm.yml
   when: ansible_distribution == "Debian"
+
+- name: Install NTP
+  apt:
+    name: ntp
+    state: present
+  when: ntp_enabled|bool
+
+- name: Start NTP Service
+  systemd:
+    name: ntp
```

Comment:
on redhat the name of the service is ntpd, while the name of the package is ntp.
should it be ntpd or ntp on debian? just wanted to make sure its not a typo :)

Reply:
I found this out when testing on ubuntu and centos. idk why the service names are different

Reply:
nice, thanks for confirming :) 
On the side note - ugh, why would they name the same thing differently?